resources/assets/templates/topic/topicDetail.hbs

```diff
@@ -1,4 +1,8 @@
+
 <div class="topic">
+    <span>
+        Current category: <a href="#categories/{{model.category.slug}}/topics">{{ model.category.name }}</a>
+    </span>
```

Comment:
Please move this block inside of topic bloc, according to mocup: below topic description, above tags